themes/wporg-translate-events-2024/blocks/components/event/excerpt.php

```diff
@@ -0,0 +1,13 @@
+<?php namespace Wporg\TranslationEvents\Theme_2024;
+use Wporg\TranslationEvents\Translation_Events;
+
+register_block_type(
+	'wporg-translate-events-2024/components-event-excerpt',
+	array(
+		// phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
+		'render_callback' => function ( array $attributes ) {
+			$event = Translation_Events::get_event_repository()->get_event( $attributes['id'] );
```

Comment:
I don't think we need to get the event from the database, we can pass the `id` attribute directly to the `get_the_excerpt()` function.